src/components/ui/Hero/hero.scss

```diff
@@ -0,0 +1,93 @@
+$media: 1280px;
+
+[data-store-hero] {
+  display: flex;
+  width: 100%;
+  background-color: var(--bg-secondary-default);
+
+  @media screen and (max-width: $media) {
+    flex-direction: column-reverse;
+  }
+
+  [data-store-hero-content] {
+    width: 100%;
+    padding: 92px var(--space-9) 92px 105px;
```

Comment:
I wonder if it's fine to have the `padding-left` to be 0 instead of 105px. It seems to decrease the width for the left-side content in favor of more space for the image (which seems proportionally larger in Figma).

![5lpiun3djs](https://user-images.githubusercontent.com/381395/147489025-372b1416-03a6-4f60-982a-43baa52b202d.gif)

Reply:
Is a good idea!